packages/client/helpers/functional-test/run-tests.ts

```diff
@@ -8,49 +12,133 @@ const allProviders = new Set(Object.values(Providers))
 const args = arg(
   process.argv.slice(2),
   {
+    // Update snapshots
     '-u': Boolean,
+    // Only run the tests, don't typecheck
     '--no-types': Boolean,
+    // Only typecheck, don't run the code tests
     '--types-only': Boolean,
+    // Restrict the list of providers
     '--provider': [String],
+    // Generate Data Proxy client and run tests using Mini-Proxy
+    '--data-proxy': Boolean,
+    // Use edge client (requires --data-proxy)
+    '--edge-client': Boolean,
+    // Don't start the Mini-Proxy server, expect it to be started externally
+    // and listening on the default port.
+    '--no-mini-proxy-server': Boolean,
+    // Don't override NODE_EXTRA_CA_CERTS. Useful if a custom mini-proxy
+    // server you start is not the one in node_modules. You then need to run
+    // `eval $(mini-proxy env)` in your shell before starting the tests.
+    '--no-mini-proxy-default-ca': Boolean,
+    // Enable debug logs in the bundled Mini-Proxy server
+    '--mini-proxy-debug': Boolean,
     '-p': '--provider',
```

Comment:
Just a style thing, could we put this one next to the `--provider` on line 22? They are tightly related but now a fair few lines apart.

Reply:
Good point